parts/k8s/windowskubeletfunc.ps1

```diff
@@ -238,6 +238,8 @@ New-NSSMService {
     & "$KubeDir\nssm.exe" set Kubelet AppDirectory $KubeDir
     & "$KubeDir\nssm.exe" set Kubelet AppParameters $KubeletStartFile
     & "$KubeDir\nssm.exe" set Kubelet DisplayName Kubelet
+    & "$KubeDir\nssm.exe" set Kubelet AppRestartDelay 5000
+    & "$KubeDir\nssm.exe" set Kubelet DependOnService docker

```

Comment:
this will need to change later once containerd is done, but ETA on that is still late this year. I'll update it then